web-test-runner-material.config.js

```diff
@@ -1,4 +1,4 @@
 /* eslint-env node */
 const { createVisualTestsConfig } = require('./wtr-utils.js');
 
-module.exports = createVisualTestsConfig('material');
+module.exports = createVisualTestsConfig('material', '88');
```

Comment:
Decided to keep Material as is to make this PR smaller. We don't actively maintain it anyway.